docs/INSTALL_GUIDE.md

```diff
@@ -0,0 +1,156 @@
+# Installation
+
+Telegraf compiles to a single static binary, which makes it easy to install.
+Both InfluxData and the community provide for a wide range of methods to install
+Telegraf from. For details on each release, view the [changelog][] for the
+latest updates and changes by version.
+
+[changelog]: /CHANGELOG.md
+
+There are wide range of places to obtain Telegraf from:
```

Comment:
```suggestion
There are a wide range of places to obtain Telegraf from:
```
The above is a grammatical fix, but thoughts on rewording to `There are many places to obtain Telegraf from:` to avoid the same wording twice in a row?